### PR TITLE
Improvement: Now app can run in background when it is manually closed…

### PR DESCRIPTION
**Pull request by TehleelMir**
… in Chinese brand devices

check if the device is a Chinese brand
        if yes, then show up a dialog that will navigate the user to the auto-start settings in his phone
        where he/she can mark this app to run in the background (because Chinese brand device don't let apps
        to run foreground notification in the background until the user manually give permission for it)

### CODE REVIEW

File: app/src/main/java/live/teekamsuthar/mutify/MainActivity.java [hunk 1]
```diff
@@ -743,4 +749,59 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
...
+    /* check if the device is chinese brand
+        if yes, then show up a dialog which will navigate user to the auto start settings in his phone
+        where he/she can mark this app to run in the background (because chinese brand device don't let apps
+        to run foreground notification in the background until user manually give permission for it)
+     */
+    private void isChineseBrand(){
+        String device = Build.MANUFACTURER.toLowerCase();
+        Intent intent;
+        switch (device){
+            case "xiaomi":
```
Comment (teekamsuthar) [inline app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:762]: Can we also add this check for Samsung devices here? I am getting frequent reports of background service getting killed from Samsung users.

Comment (TehleelMir) [reply app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:762]: sure let me update the code  and  send another pull request

**Comment by TehleelMir** [comment]
Added samsung

File: app/src/main/java/live/teekamsuthar/mutify/MainActivity.java [hunk 2]
```diff
@@ -781,6 +781,9 @@ private void isChineseBrand(){
                 intent = new Intent().setComponent(new ComponentName("com.asus.mobilemanager", "com.asus.mobilemanager.entry.FunctionActivity")).setData(
                         Uri.parse("mobilemanager://function/entry/AutoStart"));
                 break;
+            case "samsung":
+                intent = new Intent().setComponent(new ComponentName("com.samsung.android.lool", "com.samsung.android.sm.ui.battery.BatteryActivity"));
+                break;
             default:
```
Comment (teekamsuthar) [inline app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:787]: Thanks for the inclusion! Do you have any of these devices that you can test on? I only have Motorola, and I want to make sure everything's working as expected before pushing this as an update.

Comment (TehleelMir) [reply app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:787]: I have tested it on Xiaomi  and Samsung and it's working fine

Comment (TehleelMir) [reply app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:787]: Is something wrong with my Pull Request?

Comment (teekamsuthar) [reply app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:787]: Thanks for the update! I'm working on a new, more aggressive pop-up for "Device Broadcast Status" and once I finish this, I'll try to integrate your changes to that same pop up. So, the pull request will be merged soon, Thank you so much for your contribution. :)

Comment (TehleelMir) [reply app/src/main/java/live/teekamsuthar/mutify/MainActivity.java:787]: Thank You

**Pull request merged**